docs/BUSINESS.md

```diff
@@ -0,0 +1,60 @@
+---
+title: SimpleX for business
+revision: 03.12.2024
+---
+
+# Using SimpleX Chat in business
+
+SimpleX Chat (aka SimpleX) is a decentralized communication network that provides private and secure messaging. Its users are rapidly growing, and providing customer services via SimpleX can offer you a unique opportunity to engage people who are the most enthusiastic about trying out early stage technology products and services.
+
+This document aims to help you make the best use of SimpleX Chat if you choose to engage with its users.
+
+## Communcate with customers via business address
+
+In the same way you can connect to our "SimpleX Chat team" profile via the app, you can provide the address for your existing and prospective customers:
+- to buy your product and services via chat,
+- to ask any questions, make suggestions and provide feedback,
+- to discover more information about your business.
+
+Customers who value privacy and security, and want to engage with you without sharing any personal data and minimizing any metadata that is shared with you, will be really happy to use this communication channel.
+
+From v6.2 SimpleX Chat supports business addresses. Their design allows you to accept requests from multiple customers, with the app creating a new business chat with each of them.
+
+Business chats operate in a way similar to dedicated customer support systems by combining features of direct conversations and groups, and the only widely used messenger that provides such functionality is WeChat with Chinese business accounts.
```

Comment:
"... and the only widely used ..."

fact / what qualifies as widely used? add "we know of"?